docs/Reference/Reply.md

```diff
@@ -656,13 +656,14 @@ fastify.get('/json', options, function (request, reply) {
 #### Streams
 <a id="send-streams"></a>
 
-*send* can also handle streams out of the box. If you are sending a stream and
-you have not set a `'Content-Type'` header, *send* will set it at
-`'application/octet-stream'`.
+*send* can also handle streams by setting the `'Content-Type'` header to
+`'application/octet-stream'`. For proxied responses, this header should be 
```

Comment:
I don't understand the reference to proxied responses, could you clarify?

Reply:
```suggestion
`'application/octet-stream'`.
```